recipes/repository_class_design.md

```diff
@@ -0,0 +1,256 @@
+# Albums Model and Repository Classes Design Recipe
+
+_Copy this recipe template to design and implement Model and Repository classes for a database table._
+
+## 1. Design and create the Table
+
+If the table is already created in the database, you can skip this step.
+
+Otherwise, [follow this recipe to design and create the SQL schema for your table](./single_table_design_recipe_template.md).
+
+*In this template, we'll use an example table `students`*
+
+```
+# EXAMPLE
+
+Table: students
+
+Columns:
+id | name | cohort_name
+```
```

Comment:
Don't forget to update these things, or refer to the other document that includes the tables design. I also copy pasted lots of these things, so I am sure I have some mistakes as well, but I noticed quite a few words (e.g. students, albums) that were referring to previous exercises.